README.md

```diff
@@ -122,3 +122,21 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
 	}
 });
 ```
+
+## Include into project
```

Comment:
```suggestion
## Install
```

Reply:
Don't know if "Install" is the right word, maybe "Setup"?

Reply:
Setup sounds good!